pkg/satellite/satelliteweb/satelliteql/user.go

```
@@ -117,3 +118,35 @@ func fromMapUserInfo(args map[string]interface{}) (input UserInput) {
 	input.Company = fromMapCompanyInfo(companyArgs)
 	return
 }
+
+func fillUserInfo(user *satellite.User, args map[string]interface{}) satellite.UserInfo {
+	info := satellite.UserInfo{
+		Email:     user.Email,
+		FirstName: user.FirstName,
+		LastName:  user.LastName,
+		Password:  "",
+	}
+
+	for fieldName, fieldValue := range args {
+		value, ok := fieldValue.(string)
+		if !ok {
+			continue
```

We probably should return an error on invalid request.